rotkehlchen/chain/ethereum/utils.py

```diff
@@ -163,3 +169,18 @@ def should_update_protocol_cache(cache_key: GeneralCacheType) -> bool:
         key_parts=[cache_key],
     )
     return ts_now() - last_update_ts >= ETH_PROTOCOLS_CACHE_REFRESH
+
+
+def should_detect_evm_accounts(database: 'DBHandler') -> bool:
+    """Checks if the last time we detected EVM accounts is far enough to trigger
+    the process of querying it again.
+    """
```

Comment:
This function is in a strange file. Maybe it makes more sense to move it to a utils file in the tasks folder and the `EVM_ACCOUNTS_DETECT_KEY` to a constant there?

Reply:
Makes sense, thanks. I put it there because saw that `should_update_curve_pools` is there, but storing in the task manager directory perhaps makes more sense